### PR TITLE
Chinese language support

### PR DESCRIPTION
**Pull request by liuzhengjin**
Chinese language support.
In our country, many friends are using this software, so we have made multi-language support, but there may be some problems due to the large changes in the code.

### CODE REVIEW

File: src/ninjabrainbot/gui/GUI.java [hunk 1]
```diff
@@ -35,58 +35,47 @@
...
-		try {
-			font = Font.createFont(Font.TRUETYPE_FONT, Main.class.getResourceAsStream("/resources/OpenSans-Regular.ttf"));
-			fontLight = Font.createFont(Font.TRUETYPE_FONT, Main.class.getResourceAsStream("/resources/OpenSans-Light.ttf"));
-			GraphicsEnvironment ge = GraphicsEnvironment.getLocalGraphicsEnvironment();
-			ge.registerFont(font);
-			ge.registerFont(fontLight);
-		} catch (Exception e) {
-			e.printStackTrace();
-		}
-
```
Comment (Ninjabrain1) [inline src/ninjabrainbot/gui/GUI.java:86]: Why is this removed, is it incompatible with chinese characters? The text looks much worse with the default font.

Comment (Ninjabrain1) [reply src/ninjabrainbot/gui/GUI.java:86]: Okay I see why you removed it, what I dont understand is why it works fine on certain labels and on others it doesnt. For example almost all labels in the settings menu are fine with, but not in the main window. asd
I am not going to remove the better looking font for english users, but I wonder if there is a better solution than to just hard code it and say "use the default font if chinese, otherwise load this font".

File: src/ninjabrainbot/util/I18n.java [hunk 2]
```diff
@@ -16,7 +16,7 @@
  * @since 2021/12/25 14:39
  */
 public class I18n {
-    private static final ResourceBundle BUNDLE = ResourceBundle.getBundle("resources/lang/I18n", Locale.getDefault(), new UTF8Control());
+    private static final ResourceBundle BUNDLE = ResourceBundle.getBundle("resources/lang/I18n", Locale.US, new UTF8Control());
```
Comment (Ninjabrain1) [inline src/ninjabrainbot/util/I18n.java:19]: Also, how exactly does it know which language to choose, it seems to me that it will always be english based on this line.

File: src/ninjabrainbot/gui/GUI.java (hunk 1 above)
Comment (liuzhengjin) [reply src/ninjabrainbot/gui/GUI.java:86]: Because this font cannot display Chinese.
I tried to introduce Chinese fonts, but that would have made the package too big.  Maybe I can make Chinese use the default fonts of the operating system and English use the built-in fonts.

File: src/ninjabrainbot/util/I18n.java (hunk 2 above)
Comment (liuzhengjin) [reply src/ninjabrainbot/util/I18n.java:19]: I forgot to change back to the original version when I was testing My English

File: src/ninjabrainbot/gui/GUI.java (hunk 1 above)
Comment (liuzhengjin) [reply src/ninjabrainbot/gui/GUI.java:86]: Maybe I can try to solve the problem.

Comment (liuzhengjin) [reply src/ninjabrainbot/gui/GUI.java:86]: image
In the code, only the buttons on the Settings page set the font.

Comment (Ninjabrain1) [reply src/ninjabrainbot/gui/GUI.java:86]: The font is set everywhere, since ThemedLabels are used everywhere. I think the reason it works in some places is because some labels are enclosed in html tags, which may cause the text to use a default font if a characted doesnt exist in the used font. I would rather not use html tags everywhere though, I think the best solution is to just use different fonts for different languages.

Comment (Ninjabrain1) [reply src/ninjabrainbot/gui/GUI.java:86]: ill fix this

Comment (liuzhengjin) [reply src/ninjabrainbot/gui/GUI.java]: I need to go to bed because it's midnight here.

Comment (Ninjabrain1) [reply src/ninjabrainbot/gui/GUI.java]: fixed in 0eeb34d

**Comment by Ninjabrain1** [comment]
Is the en_US file necessary? The default will be used if it is absent, so it can just be deleted right?

**Comment by Ninjabrain1** [comment]
Thank you for doing this! I changed the title back to "Ninjabrain Bot" because I dont think the name should be translated, but let me know if you really think it should be.

**Comment by Ninjabrain1** [comment]
The changes will be in the next release, but until then I made this prerelease [v1.1.1-pre1](https://github.com/Ninjabrain1/Ninjabrain-Bot/releases/tag/1.1.1-pre1)

**Comment by liuzhengjin** [comment]
It is really unnecessary to translate the title.
Thank you for allowing me to add Chinese language into Ninjabrain Bot.
